contracts/solidity/transaction/Secondary.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.20;
+
+contract Secondary {
```

Comment:
nit: Looking for less general names. What about `MessageFrameAddresses` or `MessageFrameAccounts`?
Ideas?

Reply:
`MessageFrameAddresses` sounds good to me.
Thanks ... got a bit lazy with the naming I guess.